library/include/rocfft.h

```diff
@@ -210,54 +211,48 @@ ROCFFT_EXPORT rocfft_status rocfft_plan_description_set_scale_float( rocfft_plan
 ROCFFT_EXPORT rocfft_status rocfft_plan_description_set_scale_double( rocfft_plan_description description, double scale );
```

Comment:
change the scale to const here as well to be consistent?

Reply:
Sounds good; done.